pkg/orderers/segment.go

```diff
@@ -18,6 +19,32 @@ type Segment struct {
 	// List of sequence numbers for which the orderer is responsible.
 	// This is the actual "segment" of the commit log.
 	SeqNrs []t.SeqNr
+
+	// List of proposals, one per sequence number.
+	// A non-nil proposal will always be proposed by a node for the corresponding sequence number, regardless of view
```

Comment:
The comment is formulated a bit unfortunately, it should say `this node` instead of `a node`.
Different nodes can have different default proposal configuration (as is the case with checkpoints, actually).
If a value in this variable is not nil, then the node will propose it.
Will clarify.